mpas_analysis/test/test_generalized_reader.py

```diff
@@ -0,0 +1,100 @@
+"""
+Unit test infrastructure for the generalized_reader.
+
+Xylar Asay-Davis
+02/15/20167
+"""
+
+import pytest
+from mpas_analysis.test import TestCase, loaddatadir
+from mpas_analysis.shared.generalized_reader.generalized_reader \
+    import open_multifile_dataset
+
+
+@pytest.mark.usefixtures("loaddatadir")
```

Comment:
I don't know if you need this here.  It seems to me that you should get this type of functionality via use of `TestCase`.  Can you please clarify why you need `pytest` in this testing file?